crates/rspack_plugin_javascript/Cargo.toml

```diff
@@ -11,6 +11,7 @@ async-trait = { workspace = true }
 bitflags = { workspace = true }
 cow-utils = { workspace = true }
 dashmap = { workspace = true }
+either = "1"
```

Comment:
Swc uses [`either@1.10.0`](https://github.com/swc-project/swc/blob/main/Cargo.toml#L73), so we can align with swc and place it with other [swc dependencies](https://github.com/web-infra-dev/rspack/blob/main/Cargo.toml#L77-L83) for easy upgrading.